### ISSUE TITLE
Unplayable on Samsung Galaxy Tab 3 (And how we install it on Android after September 2026?)

### ISSUE THREAD

**Issue by Bastian2024** [GdGohan/SonicRacingTransformedAndroidUpdate-NewVersion#12]
When I install the game on my Samsung Galaxy Tab 3, it works, but Black Screen mostly and unplayable.

ONE THING MORE: Android is closing the unsigned APK things in September 2026, so how we play SASRT on Android 16 and 17? Galaxy a05 only handles up to One UI 7 (with Android 15)

**Comment by GdGohan** [comment]
It's an old game, and naturally new rules emerged after the game was released, it's safe
Try v4c or v4.2c

**Comment by Bastian2024** [comment]
V4c also gets flagged, and thus i can't install it anyway, it could harm my device

EDIT: wait, you said that this is safe? Even if google play protect flags it? Because i don't want a risk on my phone.

FINAL EDIT: OK, GdGohan, i should trust you, but i can't. I leave with no choice, is safe, i should trust you. Now, this issue will be closed

**Comment by Bastian2024** [comment]
I forgot that the app is built for and older.android version, so that's why google play protect blocked it and i reopened the issue, i'm on android 14 with 64-bits

**Comment by Bastian2024** [comment]
Now i noticed is an old game, now, i closed forever (no joke)

**Comment by Bastian2024** [comment]
Sorry, i had to reopen this issue to confirm if this is safe or not, if it can harm my device with android 14 64 bits (which supports installing sasrt) and if the original apk (sasasr) can run on my device android 14 with 64 bits

**Comment by Bastian2024** [comment]
But scine i think that the ssasr can run on older android such as froyo, i closing

**Comment by GdGohan** [comment]
> But scine i think that the ssasr can run on older android such as froyo, i closing

Try with VMOS or another virtual Android

**Comment by Bastian2024** [comment]
No way, instructions are for experts only.

**Comment by Bastian2024** [comment]
Qnd i want to run on my samsung a05 updated with android 14 (64 bits)

**Comment by GdGohan** [comment]
> Qnd i want to run on my samsung a05 updated with android 14 (64 bits)

Does it support 32-bit? If not, try with some virtual Android

**Comment by Bastian2024** [comment]
My system is 64 bits, so is compatable, no vm is requiered but... the issue isn't fixable, and i don't want to use a vm such as virtual android or VMOS

**Comment by Bastian2024** [comment]
And the other is, that the main 22 apk of sonic all stars racing 1.0.2 (?) Will be compitable with my device (which is android 14 with 64 bits), if is compatable but is says that is built from older android, guide me to the finish (to the fix)

**Comment by GdGohan** [comment]
New devices that already come with Android 14 do not support 32 bits

**Comment by Bastian2024** [comment]
Ok, is 64 bits, not 32 bits, now i also have another question: Can i run the Sonic Racing (Original) On my android 14 Device?

**Comment by GdGohan** [comment]
Only with a Virtual Android, if your cell phone already had Android 14 pre-installed, otherwise you just have to follow the same instructions that I had left for the Sonic Racing Transformed

**Comment by Bastian2024** [comment]
My android was beforly with android 13, now with android 14 upgraded

**Comment by GdGohan** [comment]
> My android was beforly with android 13, now with android 14 upgraded

Then follow the instruction I left

**Comment by Bastian2024** [comment]
But i said is 64 bits

**Comment by GdGohan** [comment]
> But i said is 64 bits

You said your phone was on Android 13, so follow the instructions because Android 14 limits the installation of apks with sdk lower than 23

**Comment by Bastian2024** [comment]
That you left in the SARST folder?

**Comment by GdGohan** [comment]
> That you left in the SARST folder?

V4 folder

**Comment by Bastian2024** [comment]
Is the android 14 without pc or root or just android 14???

**Comment by GdGohan** [comment]
> Is the android 14 without pc or root or just android 14???

Without pc, just Termux

**Comment by Bastian2024** [comment]
When i do all steps, the installation step fails all the time, i can't fix it, help me for an easier one

**Comment by Bastian2024** [comment]
Now that i can't make this tutorial, you should add this as "wontfix".

**Comment by GdGohan** [comment]
> When i do all steps, the installation step fails all the time, i can't fix it, help me for an easier one

Did you follow the tutorial of the video link I left in the txt? You must be in two-window mode so that the cell phone's adb does not restart

**Comment by GdGohan** [comment]
> Now that i can't make this tutorial, you should add this as "wontfix".

AGESdotJPEG tested this setup and it worked for him

**Comment by Bastian2024** [comment]
I watched the full video. The only inposible is the adb install

**Comment by GdGohan** [comment]
> I watched the full video. The only inposible is the adb install

It will only work after connecting adb in termux

**Comment by Bastian2024** [comment]
I already connected and the thing of the adb install fails everytime, is on 0.118.1

**Comment by GdGohan** [comment]
> I already connected and the thing of the adb install fails everytime, is on 0.118.1

Try 0.118.0

**Comment by Bastian2024** [comment]
But error while installing android tools, installed on 0.118.1, how i uninstall??

**Comment by GdGohan** [comment]
> But error while installing android tools, installed on 0.118.1, how i uninstall??

Uninstall the app and download 0.118.0

**Comment by Bastian2024** [comment]
I already did, installing android tools just error and normal. Not working, so i just don't want a hard tutorial

**Comment by Bastian2024** [comment]
And you should make an v4 version of the game for ANDROID 14 ONLY!

**Comment by Bastian2024** [comment]
And now i can't connect and pair the termux with my device, so make an android 14 variant

**Comment by GdGohan** [comment]
> And now i can't connect and pair the termux with my device, so make an android 14 variant

Ask to AGESdotJPEG in his channel

**Comment by Bastian2024** [comment]
But i can't enter youtube in my mobile browser

**Comment by Bastian2024** [comment]
Or is on github???

**Comment by GdGohan** [comment]
> Or is on github???

Look for his channel on YouTube or talk to him in one of the issues

**Comment by Bastian2024** [comment]
Agesdotjpg dosen't respond to me

**Comment by GdGohan** [comment]
telegram.com/@Yeah_dot_JPEG

**Comment by GdGohan** [comment]
Talk to him on his YouTube channel

**Comment by Bastian2024** [comment]
I said that i can't access youtube on mibile and i can't comment

**Comment by GdGohan** [comment]
> I said that i can't access youtube on mibile and i can't comment

Do you not have the app?

**Comment by Bastian2024** [comment]
I had the app, but i don't want to have any accidents with youtube kids videos (which i don't like the "Try Youtube Kids" thing anymore)

**Comment by Bastian2024** [comment]
So i disabled on my device

**Comment by Bastian2024** [comment]
And how i can just run sasrt and ssasr on android 14 devices (but beforly with android 13) without any protection blocks?

**Comment by GdGohan** [comment]
> And how i can just run sasrt and ssasr on android 14 devices (but beforly with android 13) without any protection blocks?

Adb instalation

**Comment by Bastian2024** [comment]
But i said that the final part (i did the video tutorial too, last time) just fails every time, You must to make an android 14 capable on the V4 folder!

**Comment by Bastian2024** [comment]
(And add v4c for notch and v4 for non-notch)

**Comment by GdGohan** [comment]
v4 supports any screen size, I will show to ages this issue

**Comment by Bastian2024** [comment]
Now you did?

**Comment by GdGohan** [comment]
yeah

**Comment by Bastian2024** [comment]
Where, in github or youtube

**Comment by GdGohan** [comment]
> Where, in github or youtube

I sent him the github link, now we just have to wait for him to help you.

**Comment by GdGohan** [comment]
> Where, in github or youtube

I sent him the github link, now we just have to wait for him to help you.

**Comment by Bastian2024** [comment]
In his issue?

**Comment by Bastian2024** [comment]
**hello?** are you here to respond?

**Comment by Bastian2024** [comment]
You didn't respond to me for **4 DAYS!!!** i having a question and you didn't respond me gdgohan, **why???**

**Comment by GdGohan** [comment]
> In his issue?

I showed it to him, he was supposed to have answered you here, and anyway, some devices after the Android 14 update, remove support for 32-bit unfortunately

**Comment by Bastian2024** [comment]
Oh, and how i can use it wothout any false flaggins of the google play portect as for older android

**Comment by GdGohan** [comment]
Disable play protect in play store temporarily

**Comment by Bastian2024** [comment]
That i was searching on the web, but i think will harm my phone by the google play protect compatibility warning

**Comment by GdGohan** [comment]
> That i was searching on the web, but i think will harm my phone by the google play protect compatibility warning

By deactivating you can install normally, then activate it and there will be no problem, mine after that no longer detected the game with this warning

**Comment by Bastian2024** [comment]
Ok  try to install the app (first, disactivate the play protect, then, install) after is installed, try to enable the play protect and scan it to check if is an false-positive

**Comment by Bastian2024** [comment]
And, how it was? **respond me!**

**Comment by GdGohan** [comment]
> And, how it was? **respond me!**

Disable in play store settings

**Comment by Bastian2024** [comment]
So when you scan it, no risk apps detected when you activated it and scanned it?

**Comment by Bastian2024** [comment]
**C'mon! Respond me!**

**Comment by GdGohan** [comment]
> So when you scan it, no risk apps detected when you activated it and scanned it?

For me, yes

**Comment by Bastian2024** [comment]
Great

**Comment by Bastian2024** [comment]
I have reopened the issue because...

If you said yes, then will say "Risk app detected"?

**Comment by Bastian2024** [comment]
Respond me!!!

**Comment by GdGohan** [comment]
No
Disable play protect, install the apk and enable play protect after that

**Comment by Bastian2024** [comment]
So no "risk app detected" warning will be ahown in the play protect page?

Just to comfirm

**Comment by Bastian2024** [comment]
**respond me as always**

Sfs (sorry for spamming)

**Comment by Bastian2024** [comment]
I need to know!

Respond me

**Comment by Bastian2024** [comment]
And i will stop spamming

**Comment by GdGohan** [comment]
> So no "risk app detected" warning will be ahown in the play protect page?
> 
> Just to comfirm

By disabling it, no, as I already explained, the game is old and has codes that do not fit the new play protect rules, but it will not have any effect.

**Comment by Bastian2024** [comment]
Got it.

Closing...

**Comment by Bastian2024** [comment]
Btw, is ARMv8

**Comment by GdGohan** [comment]
So try vmos or another virtual android

**Comment by Bastian2024** [comment]
Yeah, but i said that is only for experts and uses more storage, should i install it anyway?

**Comment by GdGohan** [comment]
Just try

**Comment by Bastian2024** [comment]
And what to do if.google play 0rotect flags it? I click on "Install anyways"?

EDIT: Can i run Orirginal Sonic and sega all stars racing on android 14 and your version of Sonic and all stars racing transformed on Android 4.4?

**Comment by GdGohan** [comment]
Of course

**Comment by Bastian2024** [comment]
Really i can install the original sonic racing version 1.0.2 on android 14.

Really?

EDIT: well, after the verification with my PIN, says "app not installed as app isn't compatible with your device" that's because i have android 13 before android 14 on my samsung galaxy a05 device

**Comment by Bastian2024** [comment]
**Respond me!!**

Plus, when i said i don't like youtube kids videos, is not, i like it, but i only watch in youtube kids, and mot on youtube original, because of "_Try YouTube Kids_" with "_An app made just for kids_" was the thing i always hate since 2023 or 2022, watching it causes something

**Comment by GdGohan** [comment]
> **Respond me!!**
> 
> Plus, when i said i don't like youtube kids videos, is not, i like it, but i only watch in youtube kids, and mot on youtube original, because of "_Try YouTube Kids_" with "_An app made just for kids_" was the thing i always hate since 2023 or 2022, watching it causes something

So try normal YouTube, why not?

**Comment by Bastian2024** [comment]
> So try normal YouTube, why not?

I already said no

I finally disable and i can't comment

**Comment by Bastian2024** [comment]
And says "App not installed as is not compatible with your device" i even have Android 13 before 14 on the Galaxy a05 and the termux one fails when finishing to install it and 0.118.0, doesn't want to install android tools

**Comment by Bastian2024** [comment]
I think Android 12 and later have a instruction to work propely in Android 12 and later, without them will crash and phone restarts, you must create a tutorial (in that issue) so that i can fix that and work Android 5.0 work propely

**Comment by Bastian2024** [comment]
OK, this issue will still remain closed

**Why?**

I already did the Virtual Master instructions and BOOM! Launches Android 5.0 VM. I'm so proud, that i will remain closed this ISSUE. So bye!

**Comment by Bastian2024** [comment]
I have to reopen it again because there's a problem
![Screenshot_2024-11-11-06-46-48](https://github.com/user-attachments/assets/85ae683e-6e6c-42ce-9762-774c5f341d81)

**Comment by GdGohan** [comment]
> I have to reopen it again because there's a problem
> ![Screenshot_2024-11-11-06-46-48](https://github.com/user-attachments/assets/85ae683e-6e6c-42ce-9762-774c5f341d81)
> 

Click on more details, and press install

**Comment by Bastian2024** [comment]
> Click on more details, and press install

VMOS + App detected + ignoring the warning = safe?

**Comment by Bastian2024** [comment]
Plus, i'm using VMOS

**Comment by GdGohan** [comment]
Yes

**Comment by Bastian2024** [comment]
OK

**Comment by Bastian2024** [comment]
@GdGohan i'm not addicted of that because can harm the VM, can be unbootable or anything else, then i need repair and then fails and i need to reset it by Deleting the OG one and making a new one :(

**Comment by GdGohan** [comment]
> @GdGohan i'm not addicted of that because can harm the VM, can be unbootable or anything else, then i need repair and then fails and i need to reset it by Deleting the OG one and making a new one :(

The game is clean, it's just a warning that the app is old.

**Comment by Bastian2024** [comment]
> The game is clean, it's just a warning that the app is old.

I don't know, an old app can get risks, there's no risks @GdGohan ?

**Comment by GdGohan** [comment]
> > The game is clean, it's just a warning that the app is old.
> 
> I don't know, an old app can get risks, there's no risks @GdGohan ?

No, this one not

**Comment by Bastian2024** [comment]
> > > The game is clean, it's just a warning that the app is old.
> > 
> > 
> > I don't know, an old app can get risks, there's no risks @GdGohan ?
> 
> No, this one not

OK... i will try later

**Comment by Bastian2024** [comment]
Is the app unsigned? @GdGohan

**Comment by Bastian2024** [comment]
Respond me

**Comment by Bastian2024** [comment]
@GdGohan 

**Hello?**

**Comment by GdGohan** [comment]
> Is the app unsigned? @GdGohan 

What do you mean? The game is signed.

**Comment by Bastian2024** [comment]
> > Is the app unsigned? @GdGohan
> 
> What do you mean? The game is signed.

Oh... just a Warning (i'm refering to the Google Play Protect's "Unsafe App Blocked" Warning)

**Comment by GdGohan** [comment]
> > > Is the app unsigned? @GdGohan
> > 
> > What do you mean? The game is signed.
> 
> Oh... just a Warning (i'm refering to the Google Play Protect's "Unsafe App Blocked" Warning)

Is it preventing you from downloading? Have you downloaded it after disabling it?

**Comment by Bastian2024** [comment]
> > > > Is the app unsigned? @GdGohan
> > > 
> > > 
> > > What do you mean? The game is signed.
> > 
> > 
> > Oh... just a Warning (i'm refering to the Google Play Protect's "Unsafe App Blocked" Warning)
> 
> Is it preventing you from downloading? Have you downloaded it after disabling it?

Nop, however, I can still play 4.2c, but i will try to download with the flag later...

**Comment by Bastian2024** [comment]
There's a tip: every APK of the SASRT is for Android 4.0 Ice Cream Sandwitch to 4.2 Jelly Bean, so that's why it was blocked by play protect, it's compatibility.

**Can you fix that?**

**Comment by GdGohan** [comment]
> There's a tip: every APK of the SASRT is for Android 4.0 Ice Cream Sandwitch to 4.2 Jelly Bean, so that's why it was blocked by play protect, it's compatibility.
> 
> **Can you fix that?**

Disable play protect before installing the apk, simple, alias test without play protect to check if everything is ok

**Comment by Bastian2024** [comment]
> > There's a tip: every APK of the SASRT is for Android 4.0 Ice Cream Sandwitch to 4.2 Jelly Bean, so that's why it was blocked by play protect, it's compatibility.
> > **Can you fix that?**
> 
> Disable play protect before installing the apk, simple, alias test without play protect to check if everything is ok

Yeah, but why you don't instead fix the problem? Every APK must be 4.0 ICS to 15 VIC (Vanilla Ice Cream), so that Google Play Protect never flag this again

**Comment by GdGohan** [comment]
It's not that simple, I would have to rewrite the entire game code and compile the lib .so again.

**Comment by Bastian2024** [comment]
> It's not that simple, I would have to rewrite the entire game code and compile the lib .so again.

Oh... google play protect whould still flag it

**Comment by Bastian2024** [comment]
And @GdGohan , i have uninstalled VMOS to free up space, now what? I'm still on Android 14, and One UI 7.0 will came out later in 2025 (One UI 7.0 haves Android 15, and One UI is exclusively for Samsung Devices, as mine is a Galaxy a05)

And again, i have to reopen the issue

**Comment by GdGohan** [comment]
> And @GdGohan , i have uninstalled VMOS to free up space, now what? I'm still on Android 14, and One UI 7.0 will came out later in 2025 (One UI 7.0 haves Android 15, and One UI is exclusively for Samsung Devices, as mine is a Galaxy a05)
> 
> And again, i have to reopen the issue

Does your phone still support 32-bit games? That don't use arm64-v8a?
I use an a05s with Android 13, and I prefer not to update either so as not to lose this support, but try with another virtual android

**Comment by Bastian2024** [comment]
> Does your phone still support 32-bit games? That don't use arm64-v8a? I use an a05s with Android 13, and I prefer not to update either so as not to lose this support, but try with another virtual android

What did i Said? I have updated to Android 14, is ARMv8, and if you want that I use VMOS, it will waste my Storage with 1GB or More

**Comment by Bastian2024** [comment]
@GdGohan 

**Respond**

**Comment by GdGohan** [comment]
> > Does your phone still support 32-bit games? That don't use arm64-v8a? I use an a05s with Android 13, and I prefer not to update either so as not to lose this support, but try with another virtual android
> 
> What did i Said? I have updated to Android 14, is ARMv8, and if you want that I use VMOS, it will waste my Storage with 1GB or More
> 
> 

I know android 14 uses arm64-v8a but some still support armeabi-v7a

**Comment by Bastian2024** [comment]
> > > Does your phone still support 32-bit games? That don't use arm64-v8a? I use an a05s with Android 13, and I prefer not to update either so as not to lose this support, but try with another virtual android
> > 
> > 
> > What did i Said? I have updated to Android 14, is ARMv8, and if you want that I use VMOS, it will waste my Storage with 1GB or More
> 
> I know android 14 uses arm64-v8a but some still support armeabi-v7a

And i cannot downgrade to Android 13, now i need to waste my storage on VMOS, and i need to work propelly without "App not installed as app isn'r compatible with your phone"

**Comment by Bastian2024** [comment]
@GdGohan 

Again, **Respond me**

**Comment by GdGohan** [comment]
> @GdGohan 
> 
> Again, **Respond me**

You didn't answer me, does your cell phone support armeabi-v7a?

**Comment by Bastian2024** [comment]
> You didn't answer me, does your cell phone support armeabi-v7a?

Yes!

But i'm on Android 14

**Comment by Bastian2024** [comment]
@GdGohan 

Once Again, **Respond**

**Comment by GdGohan** [comment]
> > You didn't answer me, does your cell phone support armeabi-v7a?
> 
> Yes!
> 
> But i'm on Android 14

Then follow the instructions I left for termux, which is in the v4 folder

**Comment by Bastian2024** [comment]
> > > You didn't answer me, does your cell phone support armeabi-v7a?
> > 
> > 
> > Yes!
> > But i'm on Android 14
> 
> Then follow the instructions I left for termux, which is in the v4 folder

But i said that the ADB Installation fails, canmot install Android Tools in Termux 0.118.0

**Comment by GdGohan** [comment]
> > > > You didn't answer me, does your cell phone support armeabi-v7a?
> > > 
> > > 
> > > Yes!
> > > But i'm on Android 14
> > 
> > Then follow the instructions I left for termux, which is in the v4 folder
> 
> But i said that the ADB Installation fails, canmot install Android Tools in Termux 0.118.0 

Try the latest version

**Comment by Bastian2024** [comment]
> > > > > You didn't answer me, does your cell phone support armeabi-v7a?
> > > > 
> > > > 
> > > > Yes!
> > > > But i'm on Android 14
> > > 
> > > 
> > > Then follow the instructions I left for termux, which is in the v4 folder
> > 
> > 
> > But i said that the ADB Installation fails, canmot install Android Tools in Termux 0.118.0
> 
> Try the latest version

But also fails, just tried with 0.118.1 before, and fails

**Comment by Bastian2024** [comment]
**Respond me please**

**Comment by GdGohan** [comment]
I don't know then, there are newer versions of termux

**Comment by Bastian2024** [comment]
> I don't know then, there are newer versions of termux

So what? Termux just fails ADB Installation, the other process works, but installation not

**Comment by GdGohan** [comment]
> > I don't know then, there are newer versions of termux
> 
> So what? Termux just fails ADB Installation, the other process works, but installation not

Well, you must keep the screen split between adb over wifi and termux because every time you exit the settings, it changes the numbers for connection

**Comment by Bastian2024** [comment]
> > > I don't know then, there are newer versions of termux
> > 
> > 
> > So what? Termux just fails ADB Installation, the other process works, but installation not
> 
> Well, you must keep the screen split between adb over wifi and termux because every time you exit the settings, it changes the numbers for connection

But i have split when doing the ADB Video, and later stull fails, wait, after i connect ADB, i still keep to split screen fot ADB Installation Process? My device, which is Samsung Galaxy a05 is 64-bits, the only option left is VMOS or other android vm?

**Comment by GdGohan** [comment]
You can use the pop up mode for the settings app and keep it until you can download the apk.

**Comment by Bastian2024** [comment]
Ok...

**Comment by Bastian2024** [comment]
Btw, i need to do the long process again, and i need a quick one, which means yeah, i need VMOS since my device is Samsung Galaxy a05, updated with Android 14, my device is 64 bits, is ARMv8, and what to do? It supports armeabi-v7a

**Comment by Bastian2024** [comment]
@GdGohan 

As always, **Respond Me**

**Comment by GdGohan** [comment]
> @GdGohan 
> 
> As always, **Respond Me**

Search for shizuku and how to use it to download apk with it on android 14, or use termux or vmos

**Comment by Bastian2024** [comment]
How to install older Android apps?

**Comment by Bastian2024** [comment]
@GdGohan where to search "shizuku"? YouTube or Google? (I have YouTube)

**Comment by GdGohan** [comment]
> @GdGohan where to search "shizuku"? YouTube or Google? (I have YouTube)

You can search for tutorials on youtube

**Comment by Bastian2024** [comment]
> > @GdGohan where to search "shizuku"? YouTube or Google? (I have YouTube)
> 
> You can search for tutorials on youtube

I don't want YouTube Kids popup risks for me, as for 202? I hated the "Try YouTube Kids, An app made just for kids" thing at all

**Comment by GdGohan** [comment]
> > > @GdGohan where to search "shizuku"? YouTube or Google? (I have YouTube)
> > 
> > You can search for tutorials on youtube
> 
> I don't want YouTube Kids popup risks for me, as for 202? I hated the "Try YouTube Kids, An app made just for kids" thing at all

You wrote that you have YouTube

**Comment by Bastian2024** [comment]
> > > > @GdGohan where to search "shizuku"? YouTube or Google? (I have YouTube)
> > > 
> > > 
> > > You can search for tutorials on youtube
> > 
> > 
> > I don't want YouTube Kids popup risks for me, as for 202? I hated the "Try YouTube Kids, An app made just for kids" thing at all
> 
> You wrote that you have YouTube

I know, but the YouTube app also haves YouTube Kids Videos and channels, which yeah, i still hatting the "Try YouTubs Kids" Thing

**Comment by Bastian2024** [comment]
@GdGohan 

Respond as i said always

**Comment by GdGohan** [comment]
> @GdGohan 
> 
> Respond as i said always

Can't watch videos through the browser? Well ask Ages here about using shizuku

**Comment by Bastian2024** [comment]
> > [@GdGohan](https://github.com/GdGohan)
> > Respond as i said always
> 
> Can't watch videos through the browser? Well ask Ages here about using shizuku

Only in Google Search

**Comment by Bastian2024** [comment]
@GdGohan

**Comment by GdGohan** [comment]
> @GdGohan 

Hi

**Comment by Bastian2024** [comment]
> > [@GdGohan](https://github.com/GdGohan)
> 
> Hi

I was asking that can i watch YouTube Videos online via Google Search

**Comment by GdGohan** [comment]
> > > [@GdGohan](https://github.com/GdGohan)
> > 
> > Hi
> 
> I was asking that can i watch YouTube Videos online via Google Search

I think you can

**Comment by Bastian2024** [comment]
> > > > [@GdGohan](https://github.com/GdGohan)
> > > 
> > > 
> > > Hi
> > 
> > 
> > I was asking that can i watch YouTube Videos online via Google Search
> 
> I think you can

Yes i can, but i cannot confirm if the video is YouTube Kids or not

**Comment by Bastian2024** [comment]
@GdGohan

**Comment by GdGohan** [comment]
> @GdGohan 

Hi

**Comment by Bastian2024** [comment]
> > [@GdGohan](https://github.com/GdGohan)
> 
> Hi

As i said i cannot comfirm is the Video is YouTube Kids or Not

**Comment by Bastian2024** [comment]
@GdGohan 

As always and ever, **Respond Me**

**Comment by GdGohan** [comment]
> @GdGohan 
> 
> As always and ever, **Respond Me**

I don't know how to answer you either and I don't know why you don't have access to normal YouTube.

**Comment by GdGohan** [comment]
> @GdGohan 
> 
> As always and ever, **Respond Me**

Try virtual master for android 14

**Comment by Bastian2024** [comment]
> > [@GdGohan](https://github.com/GdGohan)
> > As always and ever, **Respond Me**
> 
> I don't know how to answer you either and I don't know why you don't have access to normal YouTube.

@GdGohan I have Access to normal YouTube, the only thing i need is if the video is YouTube Kids or not

**Comment by Bastian2024** [comment]
> > [@GdGohan](https://github.com/GdGohan)
> > As always and ever, **Respond Me**
> 
> Try virtual master for android 14

Virtual Master? I use VMOS as for 2024

**Comment by Bastian2024** [comment]
> > > [@GdGohan](https://github.com/GdGohan)
> > > As always and ever, **Respond Me**
> > 
> > 
> > I don't know how to answer you either and I don't know why you don't have access to normal YouTube.
> 
> [@GdGohan](https://github.com/GdGohan) I have Access to normal YouTube, the only thing i need is if the video is YouTube Kids or not

Hello? I deleted YouTube, but i think that the video isn't set to YouTube Kids at least

**Comment by Bastian2024** [comment]
@GdGohan do you think that enabling that developer option solves the problem?

![Image](https://github.com/user-attachments/assets/27e26ff6-535d-4c59-a6b1-02e7556625ce)

**Comment by GdGohan** [comment]
> @GdGohan do you think that enabling that developer option solves the problem?
> 
> ![Image](https://github.com/user-attachments/assets/27e26ff6-535d-4c59-a6b1-02e7556625ce)

Maybe yes, test it

**Comment by Bastian2024** [comment]
@GdGohan I installed the app on my Samsung Galaxy Tab 3 (Android 4.2 version), but it frozes to a white screen

![Image](https://github.com/user-attachments/assets/1698d24b-1ba2-407b-9682-407956b4cbab)

Do I need to wait or install another version of the Game?

It'@ GPU is a PowerVR SGX 544MP

**Comment by Bastian2024** [comment]
@GdGohan same thing:

**Respond Me**

**Comment by Bastian2024** [comment]
Wait, do I need to use the AllGPU or the PowerVR V1 (or V2?)

**Comment by Bastian2024** [comment]
I'll try the AllGPU version (which got the WSOD Issue), but, with the OBB After installation

**Comment by Bastian2024** [comment]
Issue persists

**Comment by Bastian2024** [comment]
So I'll install the PowerVR V1 version

**Comment by GdGohan** [comment]
Try v4.2d from latest folder
Than open the game and it will ask you to download the new obb, otherwise manually install the v4 obb

**Comment by Bastian2024** [comment]
> Try v4.2d from latest folder Than open the game and it will ask you to download the new obb, otherwise manually install the v4 obb

The AllGPU version gets the White Screen, do I need to install the PowerVR V1 version or what?

**Comment by GdGohan** [comment]
Reinstall

Or install obbs manually

**Comment by Bastian2024** [comment]
> Reinstall
> 
> Or install obbs manually

I tried, and still White Screen

**Comment by Bastian2024** [comment]
Do I need to use the PowerVR V1, I tried to install the OBB Manually, White Screen on AllGPU

**Comment by Bastian2024** [comment]
@GdGohan 

Again? **Respond Me**

**Comment by GdGohan** [comment]
> Do I need to use the PowerVR V1, I tried to install the OBB Manually, White Screen on AllGPU

V4 main obb and patch obb in Android/obb/com.sega.sonic.transformed

And check if the file name is not with (1) or similar

**Comment by Bastian2024** [comment]
> > Do I need to use the PowerVR V1, I tried to install the OBB Manually, White Screen on AllGPU
> 
> V4 main obb and patch obb in Android/obb/com.sega.sonic.transformed
> 
> And check if the file name is not with (1) or similar

I doesn't have a (1) or similar, still White Screen

**Comment by GdGohan** [comment]
> > > Do I need to use the PowerVR V1, I tried to install the OBB Manually, White Screen on AllGPU
> > 
> > 
> > V4 main obb and patch obb in Android/obb/com.sega.sonic.transformed
> > And check if the file name is not with (1) or similar
> 
> I doesn't have a (1) or similar, still White Screen

Did you copy instead of cut?

**Comment by Bastian2024** [comment]
> > > > Do I need to use the PowerVR V1, I tried to install the OBB Manually, White Screen on AllGPU
> > > 
> > > 
> > > V4 main obb and patch obb in Android/obb/com.sega.sonic.transformed
> > > And check if the file name is not with (1) or similar
> > 
> > 
> > I doesn't have a (1) or similar, still White Screen
> 
> Did you copy instead of cut?

Yes, and still didn't work

I used FTP to do it

**Comment by Bastian2024** [comment]
@GdGohan Hello? Are you here to respond?

**Comment by Bastian2024** [comment]
It been **Sooo long** without answering

**Comment by Bastian2024** [comment]
@GdGohan 

Come on, **I used FTP, and It didn't work**

**Comment by GdGohan** [comment]
try vmos or VPhone OS, and install the game in it

**Comment by Bastian2024** [comment]
> try vmos or VPhone OS, and install the game in it

Is a Galaxy Tab 3, which will not work, but **it will work** on my Galaxy a05

**Comment by Bastian2024** [comment]
@GdGohan 
Hello? Waiting to a response, I need help.

Galaxy Tab 3 shows a white screen when booting it

**Comment by Bastian2024** [comment]
@GdGohan 

I have the 3DS Version of Transformed on my Homebrewed New 3DS XL

**Comment by Bastian2024** [comment]
Wait, Do I need both the Main and Patch OBB at the same com.sega.sonic.transformed?

**Comment by Bastian2024** [comment]
I solved the Galaxy Tab 3 issue, now it works!!!

**Comment by Bastian2024** [comment]
SOLUTION NEEDED: I can't see NOTHING!!!! (On a Galaxy Tab 3)

**Comment by Bastian2024** [comment]
@GdGohan I wonder, the Termux setup one will work on 64-bits Android 15?

**Comment by Bastian2024** [comment]
Hey, I solved the Issue of not working on my Galaxy a05, I used shizuku and it worked

**Comment by Bastian2024** [comment]
But the Galaxy Tab 3 Issue remains @GdGohan help

**Comment by Bastian2024** [comment]
And also, is the original Sonic & SEGA All-Stars Racing APK included in the SASRT folder is modded for newer android? It needs the OBB file too? @GdGohan

**Comment by Bastian2024** [comment]
@GdGohan can you also make Android TV ports that makes playable on both Remote Controller and Gamepad?

**Comment by Bastian2024** [comment]
Anyways, Black screen on my Galaxy Tab 3 10.1 keeps and keeps (btw is a x86 Android Tablet unlike the Galaxy Tab 3 7.0 or 8.0, which they use ARM Processors)

**Comment by Bastian2024** [comment]
@GdGohan 

And? I'm waiting to your response

**Comment by GdGohan** [comment]
Try with vphoneos

**Comment by Bastian2024** [comment]
> Try with vphoneos

Yes, but I already used Shizuku to install it on my a05, even if is Android 15.

And the thing of Tab 3 will be closed, as I removed my Firmware to install CyanogenMod 12.1 (which runs Lolipop) or to revert the stock firmware

Once I have CyanogenMod (or the Stock firmware) up, I can open this issue then

**Comment by Bastian2024** [comment]
Gonna re-open this issue because Google will close the open platform for Android on September 2026

**Comment by Bastian2024** [comment]
The date shifted to 2027, probably
